src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -25,11 +25,8 @@
          Set Private=false to avoid copying the referenced projects to output -->
     <ProjectReference PrivateAssets="all"
       Private="false"
-      Include="$(SrcLibrariesDir)\*\*.*proj;
-               $(SrcLibrariesDir)\..\ToBeMoved\*\*.*proj;
-               $(SrcLibrariesDir)\..\ToBeRemoved\*\*.*proj"
-      Exclude="$(MSBuildProjectFullPath)"
-                      />
+      Include="$(SrcLibrariesDir)\*\*.*proj"
+      Exclude="$(MSBuildProjectFullPath)" />
   </ItemGroup>
```

Comment:
The changes in this file address the post-merge feedback from @RussKie in the previous PR: https://github.com/dotnet/extensions/pull/4254#pullrequestreview-1665750137